partner-management-default.properties

```diff
@@ -67,7 +67,7 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 ## This property is used by kernel-authcodeflowproxy-api to check request is coming from allowed urls not.
-auth.allowed.urls=https://${mosip.pmp.host}/
+auth.allowed.urls=https://${mosip.pmp.host}/,https://${mosip.pmp.reactjs.ui.host}/
```

Comment:
This is for pms revamp so not required verified with @mayuradesh

Reply:
Done